### ISSUE TITLE
Category buttons.

### ISSUE THREAD

**Issue by wtf1knox** [Difegue/LANraragi#47]
Hello, I wonder, is there any options to pick category of galleries, customizable buttons like 
![2018-06-25 02_13_30-manga club read official manga for free](https://user-images.githubusercontent.com/40549904/41824693-6b40dab8-781d-11e8-93fa-70e7258cc1e4.png)

**Comment by Difegue** [comment]
You can already assign categories through tags, doing something like _category:cg set_, _category:doujinshi_, _category:manga_, etc.

Adding customizable shortcut buttons to specific tags is interesting, and wouldn't be very hard to do on my side - it's a feature of hydrus I enjoy and use a lot, so I wouldn't mind adding it in here.

Pinning this for 0.6 but I might get it in for 0.5.3 whenever I get around to releasing it (soon tm)

**Comment by ev1l0rd** [comment]
I think this is mostly in reference to sites like sadpanda.

Sites like that permit you to toggle the buttons, and having multiple on is an OR operation, not an AND, which might be a pitfall to keep in mind.

**Comment by wtf1knox** [comment]
> Pinning this for 0.6 but I might get it in for 0.5.3 whenever I get around to releasing it (soon tm)
supa nice

**Comment by Difegue** [comment]
> I think this is mostly in reference to sites like sadpanda.

Yeah I thought about the sadpanda buttons as well - It wasn't my original intention but datatables supports OR searching pretty easily through regexes: `(category:a|category:b)` does the job.  

I'll do without button toggling first to see how it all holds up, though.

**Comment by Difegue** [comment]
I'll be releasing 0.5.3 today since the changelog is big enough already - Moving this to 0.5.4.
